packages/components/src/CopyButton.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copy button that has an icon change when clicked, and copies the string passed in.
+ *
+ * props:
+ * @param {string} copy the value to copy when clicked
+ * @param {string} ButtonKind the kind of button
+ * @param {string} tooltip optional tooltip label ex. 'Copy column name'
+ * @param {string} className optional extra classname
+ * @param {string} data-testid optional extra testid
+ *
+ *
+ */
+
```

Comment:
Why'd you put the comments at the top of the file? This won't appear in Intellisense.

Instead, what you should do is put the comments above the component and the props themselves, then it'll appear. Something like:
```
type CopyButtonProps = {
  /** The value to copy when clicked */
  copy: string;
  ...
}

/**
 * Copy button that has an icon change when clicked, and copies the string passed in.
 */
const Copy Button = ({...}: CopyButtonProps): ...
```